### ISSUE TITLE
Extended Armor Bars compatibility: Mixin exception on 1.17.1

### ISSUE THREAD

**Issue by sschr15** [A5b84/armor-chroma-fabric#5]
```
InvalidInjectionException: @ModifyArg method net/minecraft/class_329::render targets a method with an invalid signature (Lnet/minecraft/class_329;Lnet/minecraft/class_4587;IIIIII), expected (Lnet/minecraft/class_4587;IIIIII) [INJECT Applicator Phase -> extendedarmorbars.mixins.json:InGameHudMixin -> Apply Injections ->  -> Inject -> extendedarmorbars.mixins.json:InGameHudMixin->@ModifyArg::render(Lnet/minecraft/class_4587;IIIIII)I from mod extended_armor_bars]
```

Simpler terms:
- A `@ModifyArg` in `InGameHudMixin` fails to inject to `InGameHud.render(InGameHud, MatrixStack, int, int, int, int, int, int)` because it's actually `InGameHud.render(MatrixStack, int, int, int, int, int, int)`

**Comment by sschr15** [comment]
Whoops I got the wrong mod sorry

**Comment by A5b84** [comment]
No you're right the mod does appear to be incompatible with Extended Armor Bars and I think it can be fixed (by displaying Armor Chroma's icons and Extended Armor Bars' toughness bar), thanks :)

Edit: links for future reference: [Extended Armor Bars](https://www.curseforge.com/minecraft/mc-mods/extended-armor-bars), [bad `@Redirect`](https://github.com/A5b84/armor-chroma-fabric/blob/1bc3e01bea8eced03c97bf33784c0863443eed05/src/main/java/nukeduck/armorchroma/mixin/InGameHudMixin.java#L29-L40)

**Comment by A5b84** [comment]
Fixed in v1.1.0
